viewer/src/utilities/networking/types.ts

```diff
@@ -47,4 +47,9 @@ export interface ModelDataClient<TModelIdentifier>
     ModelCameraConfigurationProvider<TModelIdentifier>,
     JsonFileProvider,
     BinaryFileProvider,
-    HttpHeadersProvider {}
+    HttpHeadersProvider {
+  /**
+   * Returns an identifier that can be used to identify the application Reveal is used in.
```

Comment:
Does identifier tell more than route? Why to have app Id at all when we have url tracker?

Reply:
It's the `CogniteClient`-`appId` provided on construction of the client SDK. Maybe it's a bit overlapping, but I was thinking you could have an app that had many different URLs (e.g. staging, dev, prod etc)